radiation/python/radiation_driver_gt.py

```diff
@@ -0,0 +1,1270 @@
+                Sfcprop["snoalb"],
+                Sfcprop["zorl"],
+                Radtend["coszen"],
+                Sfcprop["tsfc"],
+                tsfa,
+                Sfcprop["hprime"],
+                Sfcprop["alvsf"],
+                Sfcprop["alnsf"],
+                Sfcprop["alvwf"],
+                Sfcprop["alnwf"],
+                Sfcprop["facsf"],
+                Sfcprop["facwf"],
+                Sfcprop["fice"],
+                Sfcprop["tisfc"],
+                IM,
+                alb1d,
+                Model["pertalb"],
+            )
+
+            # Approximate mean surface albedo from vis- and nir-  diffuse values.
+            mean_surf_albedo_approx(Radtend["sfalb"], sfcalb)
+
+            lhswb = False
+            lhsw0 = True
+            lflxprf = False
+            lfdncmp = True
+
+            if nday > 0:
+
+                #  - Call module_radsw_main::swrad(), to compute SW heating rates and
+                #   fluxes.
+
+                self.rsw.create_input_data_rad_driver(
+                    plyr,
+                    plvl,
+                    tlyr,
+                    tlvl,
+                    qlyr,
+                    olyr,
+                    gasvmr,
+                    clouds,
+                    faersw,
+                    sfcalb,
+                    dz,
+                    delp,
+                    de_lgth,
+                    Radtend["coszen"],
+                    Model["solcon"],
+                    nday,
+                    idxday,
+                    IM,
+                    LMK,
+                    LMP,
+                    Model["lprnt"]
+                )
+
+                self.rsw.set_locdict_zero()
+
+                if Model["swhtr"]:
+                    self.rsw.swrad(rank=Rank)
+                else:
+                    self.rsw.swrad(rank=Rank)
+
+                Diag["topfsw"]["upfxc"][:] = self.rsw.outdict_gt4py["upfxc_t"][:,0]
+                Diag["topfsw"]["dnfxc"][:] = self.rsw.outdict_gt4py["dnfxc_t"][:,0]
+                Diag["topfsw"]["upfx0"][:] = self.rsw.outdict_gt4py["upfx0_t"][:,0]
+                Radtend["sfcfsw"]["upfxc"][:] = self.rsw.outdict_gt4py["upfxc_s"][:,0]
+                Radtend["sfcfsw"]["dnfxc"][:] = self.rsw.outdict_gt4py["dnfxc_s"][:,0]
+                Radtend["sfcfsw"]["upfx0"][:] = self.rsw.outdict_gt4py["upfx0_s"][:,0]
+                Radtend["sfcfsw"]["dnfx0"][:] = self.rsw.outdict_gt4py["dnfx0_s"][:,0]
+                scmpsw["uvbf0"] = self.rsw.outdict_gt4py["uvbf0"]#[:,0]
+                scmpsw["uvbfc"] = self.rsw.outdict_gt4py["uvbfc"]#[:,0]
+                scmpsw["nirbm"] = self.rsw.outdict_gt4py["nirbm"]#[:,0]
+                scmpsw["nirdf"] = self.rsw.outdict_gt4py["nirdf"]#[:,0]
+                scmpsw["visbm"] = self.rsw.outdict_gt4py["visbm"]#[:,0]
+                scmpsw["visdf"] = self.rsw.outdict_gt4py["visdf"]#[:,0]
+
+                transfer_values(self.rsw.outdict_gt4py["cldtausw"], cldtausw,
```

Comment:
There's a same stencil in fv3core basic_operations called `copy_defn`. It's not something we would want to swap here since this is a standalone, but we should definitely remove these duplicate stencils when we integrate radiation.